MAINTAINERS.md

```diff
@@ -143,6 +144,17 @@ git push origin <tagname>
 - [ ] Merge `master` back into `develop`.
 - [ ] Cut a new release branch from `develop`.  The branch should have the format `release/urbit-os-vX.XX`
 
+### Post-Release Deployments
```

Comment:
This is sort of pre-release? Since we're deploying the release branch before it's released